src/api/connectivity/eligibility/connectivity-eligibility.v6.service.js

```diff
@@ -1,8 +1,107 @@
-angular.module('ovh-api-services').service('OvhApiConnectivityEligibilityV6', ($resource, OvhApiConnectivityEligibility) => $resource('/connectivity/eligibility/search/buildingDetails ', {
-}, {
-  buildingDetails: {
-    method: 'POST',
-    isArray: false,
-    cache: OvhApiConnectivityEligibility.cache,
-  },
-}));
+angular.module('ovh-api-services').service('OvhApiConnectivityEligibilityV6', ($resource, OvhApiConnectivityEligibility, Poller) => {
+  const eligibility = $resource('/connectivity/eligibility', {
+  }, {
+    buildingDetails: {
+      url: '/connectivity/eligibility/search/buildingDetails',
```

Comment:
The `search` folder already exist. Why not keeping the same structure as mentionned into the README.md file?

See: https://github.com/ovh-ux/ovh-api-services/#structure

Reply:
Yes you're right, I moved this to the search folder.